### PR TITLE
Add human with meshes

### PR DESCRIPTION
**Pull request by davidegorbani**
This PR adds a urdf model of a human subject with the meshes created by @Supermonkey-design.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -25,3 +25,13 @@ install(FILES ${HUMAN_GAZEBO_FILES} DESTINATION ${CMAKE_INSTALL_PREFIX}/share/${
...
+
+file(GLOB HUMAN_WITH_MESHES humanSubjectWithMeshes/meshes/*.stl)
+
+# Install human urdf models with meshes in human-gazebo directory
+install(FILES ${HUMAN_WITH_MESHES} DESTINATION ${CMAKE_INSTALL_PREFIX}/share/${PROJECT_NAME}/meshes)
+
+set (HUMAN_WITH_MESHES_URDF humanSubjectWithMeshes/S01.urdf)
+
+# Install human urdf models with meshes in human-gazebo directory
+install(FILES ${HUMAN_WITH_MESHES_URDF} DESTINATION ${CMAKE_INSTALL_PREFIX}/share/${PROJECT_NAME})
```
Comment (traversaro) [inline CMakeLists.txt]:
```suggestion
install(FILES ${HUMAN_WITH_MESHES_URDF} DESTINATION ${CMAKE_INSTALL_PREFIX}/share/${PROJECT_NAME})

```

**Comment by traversaro** [comment]
I can't suggest it via the GitHub UI, but can you add a README.md to the HumanSubjectWithMeshes directory that reports that the models has been derived from https://blendswap.com/blend/11604 ?

**Comment by davidegorbani** [comment]
> I can't suggest it via the GitHub UI, but can you add a README.md to the HumanSubjectWithMeshes directory that reports that the models has been derived from https://blendswap.com/blend/11604 ?

I created it with https://github.com/robotology/human-gazebo/pull/35/commits/8b861489137222cc5d1dadc55dec5dc4f5adf9ff.

File: humanSubjectWithMeshes/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+The meshes in this folder has been derived from https://blendswap.com/blend/11604.
```
Comment (traversaro) [inline humanSubjectWithMeshes/README.md]:
```suggestion
The meshes in this folder have been derived from https://blendswap.com/blend/11604.
```

**Review by traversaro — APPROVED** [review]

**Review by CarlottaSartore — CHANGES_REQUESTED** [review]
Can you add in the main readme a line in the table for the human with meshes  ?  Thanks

File: README.md [hunk 3]
```diff
@@ -21,6 +21,9 @@ The measurements of each of the human subject are available from the table. Plea
 
 Currently, the legacy directory contains files related to joint motor control boards based on [gazebo-yarp-plugins](https://github.com/robotology/gazebo-yarp-plugins) and other configuration files needed to control the human joints.
 
+## Human subject with meshes
+In the folder [humanSubjectWithMeshes](./humanSubjectWithMeshes) there is a `urdf` model of a human sunject generated using the code in [human-model-generator](https://github.com/ami-iit/human-model-generator) with meshes derived from https://blendswap.com/blend/11604.
```
Comment (traversaro) [inline README.md]:
```suggestion
In the folder [humanSubjectWithMeshes](./humanSubjectWithMeshes) there is a `urdf` model of a human subject generated using the code in [human-model-generator](https://github.com/ami-iit/human-model-generator) with meshes derived from https://blendswap.com/blend/11604.
```

**Review by CarlottaSartore — APPROVED** [review]
LGTM

**Comment by claudia-lat** [comment]
Before merging @davidegorbani I think that the meshes for links and the license should be updated.

**Comment by traversaro** [comment]
@davidegorbani would you like to be mantainer here? So you can merge when you like.

**Comment by traversaro** [comment]
> @davidegorbani would you like to be mantainer here? So you can merge when you like.

You have full rights now.

**Comment by davidegorbani** [comment]
Merging!

**Pull request merged**